PyDSS/dssInstance.py

```diff
@@ -100,28 +101,35 @@ def __init__(self, params):
         if params['Frequency']['Neglect shunt admittance']:
             run_command('Set NeglectLoadY=Yes')
 
-        self._dssCircuit = dss.Circuit
-        self._dssElement = dss.Element
-        self._dssBus = dss.Bus
-        self._dssClass = dss.ActiveClass
+        self._dssCircuit = self._dssInstance.Circuit
+        self._dssElement = self._dssInstance.Element
+        self._dssBus = self._dssInstance.Bus
+        self._dssClass = self._dssInstance.ActiveClass
         self._dssCommand = run_command
-        self._dssSolution = dss.Solution
+        self._dssSolution = self._dssInstance.Solution
         self._dssSolver = SolveMode.GetSolver(SimulationSettings=params, dssInstance=self._dssInstance)
 
-        self._Modifier = Modifier(dss, run_command, params)
+        self._Modifier = Modifier(self._dssInstance, run_command, params)
         self._UpdateDictionary()
         self._CreateBusObjects()
         self._dssSolver.reSolve()
 
-        if params and params['Exports']['Log Results']:
-            if params['Exports']['Result Container'] == 'ResultContainer':
-                self.ResultContainer = RC(params, self._dssPath,  self._dssObjects, self._dssObjectsByClass,
-                                          self._dssBuses, self._dssSolver, self._dssCommand)
-            else:
-                self.ResultContainer = ResultData(params, self._dssPath,  self._dssObjects, self._dssObjectsByClass,
-                                                    self._dssBuses, self._dssSolver, self._dssCommand, self._dssInstance)
-        else:
-            self.ResultContainer = None
+        if params['Profiles']["Use profile manager"]:
+            #TODO: disable internal profiles
+            self._Logger.info('Disabling internal yearly and duty-cycle profiles.')
+            for m in ["Loads", "PVSystem", "Generator", "Storage"]:
+                run_command(f'BatchEdit {m}..* yearly=NONE duty=None')
+            self.profileStore = ProfileManager(self._dssObjects, self._dssSolver, params)
+            self.profileStore.setup_profiles()
+
+
+        #if params and params['Exports']['Log Results']:
+        if params['Exports']['Result Container'] == 'ResultContainer':
```

Comment:
Any chance we can remove this?  No worries if the answer is 'no.' ResultData can provide the same functionality as long as the user makes some changes (I think).

Reply:
I intent to remove it soon. Maybe after this merge?

Reply:
Agreed.